### ISSUE TITLE
promise rejection when typing CSS

### ISSUE THREAD

**Issue by kaylee42** [popcodeorg/popcode#875]
When typing in the CSS box, if the CSS is invalid, you get the promise rejection error seen below. If CSS is valid and you're typing, there's no promise rejection error. The app still appears to be functioning correctly when the console is closed, however it freezes popcode virtually every time you type CSS when the console is open.

<img width="1229" alt="screen shot 2017-06-11 at 4 18 23 pm" src="https://user-images.githubusercontent.com/14985351/27014196-e017d5f2-4ec1-11e7-837b-90cfc0f5043f.png">

**Comment by outoftime** [comment]
@kaylee42 good catch! want to take a crack at this? i can dig into it if not

**Comment by kaylee42** [comment]
@outoftime yeah I'd be happy to! Found this with @ibrand if you wanna work on this as well?

**Comment by ibrand** [comment]
Yeah that'd be fun, I'm in!

Let's close out that last one first though
